app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,38 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build(micropost_params)
+    @micropost.image.attach params[:micropost][:image]
+    if @micropost.save
+      flash[:success] = t ".success"
+      redirect_to root_path
+    else
+      flash[:danger] = t ".danger"
+      @feed_items = current_user.feed.page(params[:page])
+      render "static_pages/home"
+    end
+  end
+
+  def destroy
+    if @micropost.destroy
+      flash[:success] = t ".success"
+      redirect_to request.referrer || root_path
+    else
+      flash[:danger] = t ".danger"
+      redirect_to root_path
+    end
```

Comment:
```suggestion
    end
          redirect_to request.referrer || root_path
```
a đưa ra ngoài if else nhơ